platform/linux-generic/pktio/pcap.c

```diff
@@ -317,6 +322,11 @@ static int pcapif_recv_pkt(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 				if (new_pool != pcap->pool) {
 					new_pkt = odp_packet_copy(pkt, new_pool);
 
+					if (odp_unlikely(new_pkt == ODP_PACKET_INVALID)) {
+						pktio_entry->s.stats.in_discards++;
+						continue;
+					}
```

Comment:
This if-clause should be after packet_free(pkt). Seems a copy-paste error as there is a copy of the same if-clause left there.

Reply:
Copy-paste error fixed in v8.

At least in this PR, classify errors do not increment discards counter, and not all pktios have a discards counter anyway. I'm leaving that as it is for now, but maybe we'll return to this question later on.